tests/process_manager/views/test_partial_views.py

```diff
@@ -83,6 +83,41 @@ def test_get(self, auth_client):
         t2_str = t2.strftime("%Y-%m-%d %H:%M:%S")
         assert response.context["messages"][0] == f"{t2_str}: message 1"
 
+    def test_get_with_search(self, auth_client):
+        """Tests message filtering of view method."""
+        from datetime import UTC, datetime
+
+        from main.models import DruncMessage
```

Comment:
These imports are repeated in the test above so lets move them to the global scope.